src/main/java/net/minecraftforge/fluids/FluidAttributes.java

```diff
@@ -229,20 +300,27 @@ public Component getDisplayName(FluidStack stack)
 
     /**
      * A FluidStack sensitive version of getTranslationKey
+     * @param stack the provided {@code FluidStack} to get the display name of.
+     * @return Returns the translation key of the provided {@code FluidStack} as a string.
      */
     public String getTranslationKey(FluidStack stack)
     {
         return this.getTranslationKey();
     }
 
     /**
-     * Returns the translation key of this fluid.
+     * @return Returns the translation key of this fluid.
      */
     public String getTranslationKey()
     {
         return this.translationKey;
     }
 
+    /**
```

Comment:
I feel like this javadoc should be below the section comment of default accessors instead of above.